lib/app.rb

```diff
@@ -56,7 +56,6 @@ class App < Sinatra::Application
 
     SpritesCLI.new.invoke(:generate, [], :disable_optimization => true)
 
-    require 'active_support/per_thread_registry'
```

Comment:
This was added in https://github.com/freeCodeCamp/devdocs/commit/07134dd1443957e92a32ce791cdb99d80ee18d09 9 years ago to "Add missing require for active_support/cache". Might not be (probably isn't) necessary anymore.